.gitignore

```diff
@@ -3,3 +3,7 @@ td-spark*
 build/
 dist/
 *.egg-info/
+__pycache__
+
+.idea
```

Comment:
Use `.gitignore_global` for IDE/text editor specific files.

Reply:
removed d8d7541